docs/02-app/01-building-your-application/07-configuring/01-typescript.mdx

```diff
@@ -31,6 +31,8 @@ Add TypeScript to your project by renaming a file to `.ts` / `.tsx`. Run `next d
 
 If you already had a `jsconfig.json` file, copy the `paths` compiler option from the old `jsconfig.json` into the new `tsconfig.json` file, and delete the old `jsconfig.json` file.
 
+We also recommend you to use `next.config.ts` over `next.config.js` for better type inference.
```

Comment:
We can link to this section - just need to double check this is the right hash link. 

```suggestion
We also recommend you to use [`next.config.ts`](#type-checking-with-nextconfigts) over `next.config.js` for better type inference.
```

Reply:
> just need to double check this is the right hash link
Confirmed on VS Code `Markdown Preview`!